src/components/CookieBanner.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+import { useCookies } from 'react-cookie';
+import Button from './Button';
+import Modal from './Modal';
+import { Switch } from '@headlessui/react';
+import { LockClosedIcon } from '@heroicons/react/outline';
+import CookieSettings from './CookieSettings';
+
+export const COOKIES_ACCEPTED = 'holaplex_cookies_acceptance';
+export const ANALYTICS_ACCEPTED = 'holaplex_analytics_cookies_acceptance';
+export const PREFERENCE_ACCEPTED = 'holaplex_preference_cookies_acceptance';
```

Comment:
should these be exported from e.g. the views dir instead of in the component (thinking about the direction of dependencies)?

Reply:
I think they're fine as part of the component for now if it becomes an issue i'll move them, keeping cookie related stuff together seems like the best pattern atm